packages/flutter_tools/lib/src/commands/update_packages.dart

```diff
@@ -23,9 +23,6 @@ const Map<String, String> _kManuallyPinnedDependencies = <String, String>{
   // Add pinned packages here.
   'flutter_gallery_assets': '0.1.9+2', // See //examples/flutter_gallery/pubspec.yaml
   'mockito': '^4.1.0',  // Prevent mockito from downgrading to 4.0.0
-  'test': '1.6.3',         //  | Tests are timing out at 1.6.4
```

Comment:
Shouldn't there be upgrades to this package?

Reply:
It looks like we have other dependency constraints that are keeping us from upgrading...